### ISSUE TITLE
Staging Review finding: Continuity of Care and Self-Entered Health Information download links not keyboard accessible

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99391]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

This ticket was part of a merged Staging Review with MHV Medical Records [Download Records](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92340), [Flow/Images](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95081) and [Settings](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95082)

## Product Information
**Team:** vfs-mhv-medical-records
**Product:** MHV Medical Records
**Feature:** Phase 1 Radiology Flow/Images, Download Records, and Settings

## Findings details
**VA.gov Experience Standard - issue:** [User can't complete a necessary flow or hits a dead end.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/medical-records/download](https://staging.va.gov/my-health/medical-records/download), the download links for the Continuity of Care file ("Download .xml file") and the self-entered health information file ("Download PDF") do not receive focus when tabbing through interactive elements with your keyboard. There is no way for a keyboard-only user to download these files.

<details><summary>Link, screenshot or steps to recreate</summary>

![Expanded accordions for Continuity of Care (with a "Download .xml file" link) and Self-entered health information (with a "Download PDF" link).](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/92340-Brian-2.png?raw=true)

</details>

## Recommended action

The links don't have an `href` attribute, which is what signals to browsers that they should be focusable. There must be an `href` attribute on all links, even a perfunctory `#` if there's some kind of onClick event.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 2.1.1 Keyboard - Level A](https://www.w3.org/TR/WCAG21/#keyboard)
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by bryan-riley-va** [comment]
We have resolved the issue (MHV-65354).  This was actually resolved as part of MHV-65229 as well.  Screenshots of the validation in Staging are below.

https://jira.devops.va.gov/browse/MHV-65354


![Image](https://github.com/user-attachments/assets/d211294a-ee6c-45a8-a09c-0bb8066338a3)
![Image](https://github.com/user-attachments/assets/ecb380b9-122f-447f-8d8d-b7fa97ab5781)
